### DISCUSSION TITLE
Help needed in editing my file name

### DISCUSSION THREAD

**Discussion by ghost** [community/community#24197] [Mobile]
<p>I couldn’t change the name of my file through my Android phone</p>

**Comment by sirlupinwatson** [comment, marked as answer]
<p>Hi <a class="mention" href="/turbo5411n">@turbo5411n</a> ,</p>
<p>You can try to update your browser apps on your mobile phone. You can also use the Desktop version from your cellphone. </p>
<p>Look at the top of your web page, Options, and scroll down to see a box that you can Check / Uncheck . Mark it Desktop site and then reload the page. </p>
<p>Sometimes the first click is not working, you will still see the mobile version, click again on a link, and it should work. </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/a3db6e5729731205f5d39f43353d8d640c3aba5a.jpeg" data-download-href="/uploads/short-url/nnxXiFzjiaIq4YzQvDCuvAcpOyu.jpg?dl=1" title="11111.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/55941045/181083245-a0d20168-3c06-4d31-b711-08f16d6633d8.jpg" alt="11111.jpg" data-base62-sha1="nnxXiFzjiaIq4YzQvDCuvAcpOyu" width="243" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/a/a3db6e5729731205f5d39f43353d8d640c3aba5a_2_243x500.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/a/a3db6e5729731205f5d39f43353d8d640c3aba5a_2_364x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/a/a3db6e5729731205f5d39f43353d8d640c3aba5a_2_486x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/a3db6e5729731205f5d39f43353d8d640c3aba5a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">11111.jpg</span><span class="informations">1080×2220 181 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I hope it can help you !</p>
<p>Have a good evening !</p>
